static/css/zamboni/zamboni.css

```diff
@@ -2405,6 +2404,10 @@ form .error .note.error {
     font-size: 1.05em;
 }
 
+.other-categories.highlight ul:nth-of-type(1) {
```

Comment:
Is there a need to only do this for the first one?

As a side-note using `:first-of-type` might be a more direct way to do the same thing.